### DISCUSSION TITLE
Lodestar Standup 2024-2 (January 16, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6274] [Standups]
### Meeting Info

[Jan 16, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/jan-16-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.14.0 Release
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
I would like to add discussion on https://github.com/ChainSafe/lodestar/pull/6033 and any potential alternatives solutions if we're unable to reach someone from leveldb to unblock this.

**Comment by twoeths** [comment]
I'd like to discuss the plan to increase peer count to 100 https://github.com/ChainSafe/lodestar/issues/5799

**Comment by philknows** [comment]
I think we should include the discussion on how we want to go ahead with the self hosted runners as commented here: https://github.com/ChainSafe/lodestar/pull/6282#issuecomment-1888746509

**Comment by philknows** [comment]
Would also like to quickly discuss opinions on this comment about some older portions of the validator configuration docs: https://github.com/ChainSafe/lodestar/pull/6295#discussion_r1452580536

**Comment by philknows** [comment]
Closed in favour of #6313 .

Notes are here: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#january-16-2024-planning-and-standup-meeting
